### ISSUE TITLE
[Bug]: Use "LRM" (location referencing method) and not "LRS" (location referencing system)

### ISSUE THREAD

**Issue by vlcinsky** [DATEX-II-EU/DatexII#532]
### Bug description

In our class model for location referencing, we currently (as of D2 v3.x) use:
- LRS
- Location referencing system

This does not respect the distinction between Location Referencing **System** (sort of implementation or even deployment) and Location Referencing **Method** (sort of algorithm)

Another issue is, that (at least for me), the abbreviations LRM and LRS are well established in context of location referencing without being bound only to linear locations.

# Definitions in other standards

## ISO 17572-1:2022 ITS - Location referencing for geographic databases Part 1: General requirements and conceptual model

- **location referencing system (LRS)**: complete system by which location references are generated, according to a location referencing method, and communicated, including standards, definitions, software, hardware, and databases

- **location referencing method (LRM)**: methodology of assigning location references to locations

Note, that the LRS talks about "software, hardware, databases" and this is definitely not we deal with in our class model. We deal with pure LRM (method) and location references (serialized referenced to a location).

## ISO 19148:2024 Geographic information — Linear referencing

- **linear referencing method (LRM)**: manner in which measurements are made along (and optionally offset from) a linear element

- **linear referencing system (LRS)**: set of linear referencing methods (3.11) and the policies, records and procedures for implementing them

# Discusion

In both standards, there is distinction between "method" (sort of pure algorithm) and "system" (sort of implementation or even deployment).

I am aware of the fact, that in EN 16157-2 we already use abbreviation LRM for Linear Referencing method which apparently comes from ISO 19148.

Anyway, our EN 16157-2 does not deal only with linear locations. It covers multiple location referencing methods (AlertC, OpenLR and many more).

# Proposed action

Accept the fact, we do not work with (location referencing) systems but methods.

Keep the abbreviation LRM for Location Referencing Method (and update reference to relevant standard ISO 17572-1)

In case we need to use terms and concepts from ISO 19148 (we probably do):
- review proper distinction between system and method
- if there are conflicting abbreviations, stop using these and prefer the (more general) ones from ISO 17572-1

### Version

4

**Comment by LBlaive** [comment]
The mentioned standards are known, especially ISO 17572-1 where I was involved in the corresponding working group (ISO/TC 204/WG 3) at the initial version. It is the reason I use the expression "Location referencing system" and not "location referencing method". Indeed this ISO standard defined first two LRMs: 

- one about "Pre-coded location references" (ISO 17572-2) that includes systems like Alert-C, VICS etc.;
- one about "Dynamic location references" (ISO 17572-3) that primarily standardised Agora-C but also covers OpenLR or TPEG-Loc.
- A third one named "Precise relative location references" was added in 2018.

Regarding EN ISO 19148 (prepared by ISO/TC 211/WG 10) the definition is slightly different but I do not see any fundamental contradiction.
In conclusion and to remain simple, I prefer to use the expression "location referencing system (and the acronym LRS) for anything that is defined in EN 16157-2. Datex II does not deal with methods or at an abstract conceptual level but focuses on exchanges and defines concrete data, which falls in a system.
